### PR TITLE
chore: SkiaSharpTest Uno 4.1.9

### PR DESCRIPTION
**Pull request by iury-kc**
SkiaSharpTest updated to Uno 4.1.9

Last stable SkiaSharp doesn't work. So, used the last preview.

### CODE REVIEW

**Comment by iury-kc** [comment]
Last stable SkiaSharp doesn't work. So, used the last preview.

File: UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml [hunk 1]
```diff
@@ -21,13 +21,12 @@
 			<skia:SKSwapChainPanel
 				x:Name="swapChain"
 				PaintSurface="OnPaintSwapChain"
-				PointerMoved="{x:Bind OnPointerMoved}"
-				Background="Transparent"
+				PointerMoved="OnPointerMoved"
```
Comment (jeromelaban) [inline UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml:24]: Why did you remove the x:Bind ?

Comment (iury-kc) [reply UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml:24]: Wasn't working. I raises method/event not found. 
Probably because the SkiaSharp package upgrade.

Comment (iury-kc) [reply UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml:24]: Hummm! It was raising this:
System.Exception
  Message=Failed to load Page SkiaSharpTest.MainPage
 It was necessary to Droid for example, to make it works.

However, i tested the Bind now with the preview version (the one I used to make WASM work correctly and the Bind is working perfectly. Downgrading to 2.80.3 will make it not work. 

Will make a squash now, undoing the Bind changes.

File: UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml [hunk 2]
```diff
@@ -21,8 +21,7 @@
 			<skia:SKSwapChainPanel
 				x:Name="swapChain"
 				PaintSurface="OnPaintSwapChain"
-				PointerMoved="{x:Bind OnPointerMoved}"
-				Background="Transparent"
```
Comment (jeromelaban) [inline UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml:25]: You must keep the transparent background, otherwise on some platforms the pointer movements will not work.

Comment (iury-kc) [reply UI/SkiaSharpTest/SkiaSharpTest/SkiaSharpTest.Shared/MainPage.xaml:25]: I was the sampe reason, the stable version wasn't accepting this property. The preview does. Now it's restored.

**Review by jeromelaban — APPROVED** [review]
